ansible/roles_ocp_workloads/ocp4_workload_windows_node/tasks/workload.yml

```diff
@@ -253,6 +253,26 @@
   until:
     - windowsjob.resources[0].status.startTime is defined
 
+# Create the winc-ssh that allows for ssh into the Windows node
+- name: Create the winc-ssh that allows for ssh into the Windows node
+  k8s:
+    state: present
+    definition: "{{ lookup('template', 'winc-deploy.j2' ) }}"
+
+# Wait for the winc-ssh to come up
+- name: Wait for the winc-ssh to come up
+  k8s_info:
+    api_version: apps/v1
+    kind: Deployment
+    name: winc-ssh
+    namespace: "{{ ocp4_workload_windows_node_namespace }}"
+  register: wincssh
+  retries: 50
+  delay: 10
+  until:
+    - wincssh.resources[0].status.readyReplicas is defined
+    - wincssh.resources[0].status.readyReplicas == 1
```

Comment:
I usually compare to .spec.replicas. Then it works for more than 1 as well. But this is fine.